bucket/ugrep.json

```diff
@@ -1,21 +1,26 @@
 {
-    "version": "4.3.6",
+    "version": "4.4.1",
     "description": "A faster, user-friendly and compatible grep replacement",
     "homepage": "https://ugrep.com/",
     "license": "BSD-3-Clause",
-    "url": "https://github.com/Genivia/ugrep/releases/download/v4.3.6/ugrep.exe#/ug.exe",
-    "hash": "99327decad4a0f68f7037540233e82fff70ca0db95ae34fe8ee143ea6581ef9a",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Genivia/ugrep/releases/download/v4.4.1/ugrep-windows-x64.zip",
+            "hash": "68f90eb538fbd024979a5e575d7a9be4501aa8df3f95fd08ac4a1f51a66bd405"
+        }
+    },
     "bin": [
-        "ug.exe",
-        [
-            "ug.exe",
-            "ugrep"
-        ]
+        "ugrep.exe",
+        "ug.exe"
```

Comment:
> The ug command is intended for context-dependent interactive searching and is equivalent to the `ugrep --config --pretty --sort` command to load the default configuration file `.ugrep' when present in the working directory or in the home directory.

We can shim `ug` to `ugrep --config --pretty --sort`, but since there is an extra executable why not just use it 🤔

Reply:
The programs change its behaviour with the name of the executable file? I did not know about that, I just noticed that they are same hash.

Reply:
Yes, it checks the actual executable name and not the shim which means that the previous `ug` -> `ugrep` shim was incorrect (it would still act as `ug`).